js/ui/overlay/ui.overlay.js

```diff
@@ -334,7 +334,7 @@ const Overlay = Widget.inherit({
     },
 
     _contentResizeHandler: function(entry) {
-        if(!this._shouldSkipContentResize(entry)) {
+        if(!this._shouldSkipContentResize(entry) && this._$content.is(':visible')) {
```

Comment:
Isn't it better to use our api to check if popup is opened?

Reply:
No, the parent container is hiding. So, visibilityChange for nested component doesn't call and ValidationMessage stay visible. It's important check visibility this way.